api/app.go

```diff
@@ -99,6 +126,35 @@ func (app *App) Configure() error {
 		return err
 	}
 
+	//we are customizing the default http error reply
+	runtime.HTTPError = func(ctx context.Context, mux *runtime.ServeMux, marshaler runtime.Marshaler,
+		w http.ResponseWriter, _ *http.Request, err error) {
```

Comment:
nit: this line break looks weird, as the parameters are at the same level as the body.

Could you try something like this (and `goimports -w` it)

```golang
	runtime.HTTPError = func(
			ctx context.Context, mux *runtime.ServeMux, marshaler runtime.Marshaler,
			w http.ResponseWriter, _ *http.Request, err error) {
		// body at this level
	}
```

Reply:
I placed all parameters on a single line. go imports wasn't playing along and kept moving the parameters lines to the same indentation of the function body.